Sources/NIO/Bootstrap.swift

```diff
@@ -575,17 +586,29 @@ public final class ClientBootstrap: NIOClientTCPBootstrapProtocol {
         }
     }
 
+    #if !os(Windows)
+        /// Use the existing connected socket file descriptor.
+        ///
+        /// - parameters:
+        ///     - descriptor: The _Unix file descriptor_ representing the connected stream socket.
+        /// - returns: an `EventLoopFuture<Channel>` to deliver the `Channel`.
+        @available(*, deprecated, renamed: "ClientBoostrap.withConnectedsocket(socket:)")
+        public func withConnectedSocket(descriptor: CInt) -> EventLoopFuture<Channel> {
+          return self.withConnectedSocket(socket: descriptor)
+        }
+    #endif
+
     /// Use the existing connected socket file descriptor.
     ///
     /// - parameters:
     ///     - descriptor: The _Unix file descriptor_ representing the connected stream socket.
     /// - returns: an `EventLoopFuture<Channel>` to deliver the `Channel`.
-    public func withConnectedSocket(descriptor: CInt) -> EventLoopFuture<Channel> {
+    public func withConnectedSocket(socket: NIOBSDSocket.Handle) -> EventLoopFuture<Channel> {
```

Comment:
the word socket is duplicated. This should be `withConnectedSocket(_ socket: ...)`